sdk/communication/communication-sms/tests.yml

```diff
@@ -5,8 +5,6 @@ stages:
     parameters:
       PackageName: "@azure/communication-sms"
       ServiceDirectory: communication
-      MatrixFilters:
-        - DependencyVersion=^$
```

Comment:
What does this setting do?

Reply:
I believe this was added a while back to disable the min-max tests. So now that we're re-enabling them this should work.
@KarishmaGhiya can you please confirm?

Reply:
I'm going to dig into this again

Reply:
looks like that is indeed what this removal does. you can see in the test runs it is running (and passing!) min max tests

Reply:
Yes that is right, this line was added as a filter on "Dependency Version" to disable min-max tests. But now you want to enable them, so we should remove this filter.